src/components/Home/Header/index.tsx

```diff
@@ -47,6 +60,83 @@ const logosData: Array<{
   widthLg: number
   alt: string
 }> = [
+  {
+    widthSm: 66,
+    widthMd: 88,
+    widthLg: 136,
+    src: fastapiLogo,
+    alt: 'FastAPI logo'
+  },
```

Comment:
@iterative/websites, could you please help to understand how I can make these images lead to redirect to a specific docs page? E.g. here I want a link to `/doc/user-guide/serving/fastapi`. For each logo the link will be different.

Reply:
Sure, we can simply wrap <Img> tag with anchor tag <a>. But, I guess we would want some styling to make sure it shows it's clickable. I will set one up for you. I guess I will be adding add another field `link` in the above data and it will make the logo clickable.